cmd/snap-seccomp/main.go

```diff
@@ -789,28 +795,133 @@ func complainAction() seccomp.ScmpAction {
 	return seccomp.ActAllow
 }
 
+var osCreateTemp = os.CreateTemp
+
+func exportBPF(fout *os.File, filter *seccomp.ScmpFilter) (bpfLen int64, err error) {
+	// TODO: use a common way to handle prefixed errors across snapd
+	errPrefixFmt := "cannot export bpf filter: %w"
+
+	oldPos, err := fout.Seek(0, os.SEEK_CUR)
```

Comment:
```suggestion
	oldPos, err := fout.Seek(0, os.SeekCurrent)
```

Reply:
It should be io.SeekCurrent, right?

Reply:
Done in 02c8a9ff81a8f036714eb4c0afce940cfa7c3c01